sdk-extensions/autoconfigure/src/main/java/io/opentelemetry/sdk/autoconfigure/TracerProviderConfiguration.java

```diff
@@ -152,6 +154,20 @@ static Sampler configureSampler(String sampler, ConfigProperties config) {
           }
           return Sampler.parentBased(Sampler.traceIdRatioBased(ratio));
         }
+      case "jaeger":
```

Comment:
Is this an official option in the opentelemetry specification? I don't see it here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/sdk-environment-variables.md#general-sdk-configuration

Reply:
It's not. The proposal is to create a new option `jaeger` to be able to fetch the sampling configuration from the Jaeger Collector. Is there a problem to create a new option?

Reply:
We would want to have it in the official specification before implementing in java specifically, as we want to make sure that all languages use the same options, formats, names, etc.

Reply:
Makes sense. I will close this PR